src/components/step-nav.scss

```diff
@@ -98,21 +100,24 @@ $StepNav-border--small: 2px;
   position: relative;
   z-index: 10;
   color: $ui-divider;
+  padding: 0 $StepNav-border;
 }
 
 .StepNav-stepTitle {
   display: block;
   text-align: center;
   margin: 0 auto;
   color: $ui-divider;
-  padding-top: $StepNav-border;
   font-family: $base-font-family;
   font-size: $heading-size--small;
+  position: absolute;
+  width: $StepNav-titleWidth;
+  bottom: -20px;
```

Comment:
I wonder if doing something like `top: 100%` and building some padding into the title might be less fragile in case the text breaks into two lines:
 
<img width="163" alt="screen shot 2017-06-12 at 10 35 34 am" src="https://user-images.githubusercontent.com/9888467/27046920-1a03b45e-4f5b-11e7-823d-0c6aa643afee.png">